packages/broker/src/plugins/storage/StorageConfig.ts

```diff
@@ -49,7 +30,7 @@ export class StorageConfig {
 
     static ASSIGNMENT_EVENT_STREAM_ID_SUFFIX = '/storage-node-assignments'
 
-    streamKeys: Set<StreamKey>
+    private spidKeys: Set<Protocol.SPIDKey>
```

Comment:
Again, wonder whether it would be more clear to use full name?
```suggestion
    private streamPartitions: Set<Protocol.SPIDKey>
```